README.md

```diff
@@ -1,283 +1,13 @@
-# Golem Rust
+# golem-rust
 
-This crate contains couple of Rust macros that facilitate writing Golem Cloud backends in Rust:
-1. Derives `From<>` and `Into<>` typeclasses between wit-bindgen derived data types and custom domain model data types.
-2. Generates wit file from rust code.
```

Comment:
Just out of curiosity. So this PR is removing these features and just exposing transactional host functions? Or is it a mix of different things that help users write golem backend?

Reply:
Ok, i think I got the answer

Reply:
Just moved the macro into `golem-rust-macro`.